automation/alpha-perf-test.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+# Simple script to start perf test in alpha
+echo "Starting Perf test in alpha"
+echo "FOR EASE OF USE: IF YOU RUN THIS SCRIPT AND A BROWSER OPENS WITH GOOGLE LOGIN, KILL THE SCRIPT AND GCLOUD AUTH INTO EACH USER FIRST (see script comments)"
+
+# Log in with each user:
+# gcloud auth login dominique.testerson@gmail.com
+# gcloud auth login gary.testerson1@gmail.com
+# gcloud auth login felicity.testerson@gmail.com
+# gcloud auth login frida.testerson@gmail.com
+# gcloud auth login elvin.testerson@gmail.com
+# gcloud auth login test.firec@gmail.com
+
+
+echo "Logging in with dominique.testerson@gmail.com"
+gcloud auth login dominique.testerson@gmail.com
+ACCESS_TOKEN=$(gcloud auth print-access-token)
+curl 'https://firecloud-orchestration.dsde-alpha.broadinstitute.org/api/workspaces/perf-test-a/Perf-test-A-workspace/submissions' -H 'origin: https://firecloud.dsde-alpha.broadinstitute.org' -H 'accept-encoding: gzip, deflate, br' -H "authorization: Bearer $ACCESS_TOKEN" -H 'content-type: application/json' --data-binary '{"methodConfigurationNamespace":"qamethods","methodConfigurationName":"sleep1hr_echo_strings","entityType":"sample_set","entityName":"sample_set6k","useCallCache":false,"expression":"this.samples"}' --compressed
```

Comment:
Seems like it might be nice if you parameterized the environment as an argument, but up to you.

Reply:
This is using Alpha users and billing projects/workspaces, not sure if applicable to any other env.

Reply:
Fair enough, never mind then!